Makefile.machine.include

```diff
@@ -56,8 +56,8 @@ CL_MANYCORE_HOST_COORD_Y             := 1
 CL_MANYCORE_DIM_X                    := $(BSG_MACHINE_GLOBAL_X)
 CL_MANYCORE_DIM_Y                    := $(shell expr $(BSG_MACHINE_GLOBAL_Y) - 1)
 
-CL_MANYCORE_MEM_CFG                  :=e_vcache_blocking_axi4_f1_model
-CL_MANYCORE_DRAM_CHANNELS            :=1
+CL_MANYCORE_MEM_CFG                  := e_vcache_blocking_test_dramsim3_hbm2_4gb_x128
```

Comment:
can we drop _4gb_x128, since we will not be using other configs.

Reply:
I think we should leave it. We might use another config in the future.